app/src/main/java/fr/free/nrw/commons/nearby/PlacesLocalDataSource.java

```diff
@@ -36,4 +36,11 @@ public Place fetchPlace(String entityID){
     public Completable savePlace(Place place) {
         return placeDao.save(place);
     }
+
+    /**
+     * Deletes a Place object from the database
+     * based on the provided entity.
+     */
+    public void  deletePlace(final Place place) {placeDao.deletePlace(place);}
```

Comment:
nitpick: double space, please make it single space

Reply:
Sure will do ! would you want me to add this as a new commit or revert and edit the previous commits ?

Reply:
New commit is fine, before merging I will squash all of this branch's commits into a single commit anyway. :-)

Reply:
Thank you, I have pushed the changes.